presidio-api/cmd/presidio-api/main.go

```diff
@@ -124,6 +125,25 @@ func setupHTTPServer(port int, loglevel string) {
 			projects.POST("/schedule-streams-job", scheduleStreamJob)
 		}
 
+		// recognizers group
+		// /api/v1/analyzer/recognizers
+		recognizers := v1.Group("/analyzer/recognizers")
+		{
+			// Get an existing recognizer
+			recognizers.GET(":id", getRecognizer)
```

Comment:
The REST -> GRPC mapping could be handled by a [GRPC-Gateway](https://github.com/grpc-ecosystem/grpc-gateway), however, I've never used this and would say it's out of scope of this PR. 

It would be good to track this as an issue and maybe spike to see if this can be done.

Reply:
thanks, added as a new task